Sources/Fuzzilli/Compiler/Parser/parser.js

```diff
@@ -519,6 +519,15 @@ function parse(script, proto) {
                 let argument = visitExpression(node.argument);
                 return makeExpression('UnaryExpression', { operator, argument });
             }
+            case 'ConditionalExpression': {
+
```

Comment:
Nit: maybe remove this blank line

Reply:
Here I also removed two more blank lines such that this case statements style aligns with the rest of the code. The other comments below this one have also been addressed with commit 3b71e9fab14ae4b48bf87b3817064dfe823e1091